Mathlib/Analysis/RCLike/Basic.lean

```diff
@@ -39,7 +40,7 @@ their counterparts in `Mathlib/Analysis/Complex/Basic.lean` (which causes linter
 A few lemmas requiring heavier imports are in `Mathlib/Data/RCLike/Lemmas.lean`.
 -/
 
-open scoped ComplexConjugate
+open scoped BigOperators ComplexConjugate
```

Comment:
```suggestion
open scoped ComplexConjugate
```
somebody told me that's a no-op :)

Reply:
Not anymore! `expect` is scoped to it now